mocks/mocks.py

```diff
@@ -467,6 +467,15 @@ def mock_image_stack(input_image,
     """
     # measuring the time for stacking images.
     start_time = time.time()
+    try:
+        exptime = exptime.to(u.second)
+    except u.UnitConversionError:
+        message = "The exposure time must have dimension of time."
+        raise u.UnitsError(message)
+    except AttributeError:
+        message = "While the exposure time should be a quantity with \
+dimension of time, it is a dimensionless {}".format(type(exptime))
+        raise AttributeError(message)
```

Comment:
Also, I think a new function can do the whole process that has been added here and it could be located at utils. Please advise me about that as well.

Reply:
@bazkiaei what you might do with this is make a fork/branch/PR in gunagala for a new function located where `ensure_units` is and call it `require_units`. That'd be useful I think as a second option where you really do need the units.